docs/tutorials/projects/filtered-camera.md

```diff
@@ -91,7 +91,10 @@ Once you have added the data management service, enable image data capture for y
      {{< imgproc src="/tutorials/filtered-camera-module/configure-webcam-data-capture.png" alt="The camera component configuration pane in the Viam app with data capture configured and enabled" resize="800x" >}}
 
 1. Click **Save** in the top right corner of the page to save your changes.
-1. In the [Viam app](https://app.viam.com), navigate to the [**DATA**](/data/view/) tab, where you should see images captured by your camera component appearing steadily.
+1. You can check that data is being captured by clicking on the menu on the camera and selecting **View captured data**.
```

Comment:
opt nit: phrasing:

```suggestion
1. You can check that data is being captured by clicking on the menu icon on the camera configuration pane and selecting **View captured data**.
```

Or similar. Just to avoid on the ... on the